source/administration/incident-response-application.rst

```diff
@@ -0,0 +1,466 @@
+
+Add Users to Channel (type: ``add_users_channel``)
+**************************************************************
+
+The add users to channel action adds the specified users to the channel.
+
+.. csv-table::
+    :header: "Field", "Description", "Type", "Required"
+
+    "channel_name", "The channel to create the post in. Can be name or ID.", "model.Channel", "Yes"
+    "users", "Users to add to the channel after creation. Can be usernames, user IDs or AD/LDAP group names.", "[]string, "Yes"
+
+// TODO Verify JSON schema and add an example here.
+
+Create Post (type: ``post``)
+*******************************
+
+Creates a post in the specified channel.
+
+.. csv-table::
+    :header: "Field", "Description", "Type", "Required"
+
+    "channel_name", "The channel to create the post in. Can be name or ID.", "string", "Yes"
+    "message", "The contents of the message.", "string", "Yes"
+    "fields", "A list of fields to show in the message. ", "[]Field", "No"
+    "transition_button", "A label and a step to transition to when pressed.", "[]Button", "No"
+
+// TODO Verify JSON schema and add an example here. Example should include fields and transition_button.
+
+Archive Channel (type: ``archive_channel``)
+**********************************************
+
+Archives the specified channel.
+
+.. csv-table::
+    :header: "Field", "Description", "Type", "Required"
+
+    "channel_name", "The channel to archive. Can be name or ID.", "string", "Yes"
+
+// TODO Verify JSON schema and add an example here.
+
+Transition to Another Step (type: ``transition_to``)
+******************************************************
+
+Specified which step to transition the workflow to. // TODO Add a more clear description
+
+.. csv-table::
+    :header: "Field", "Description", "Type", "Required"
+
+    "to", "The name of the target step to transition to", "string", "Yes"
+
+// TODO Verify JSON schema and add an example here.
+
+Statistics
+^^^^^^^^^^^^^^^
+
+The incident response application also enables you to pull summary statistics, including mean-time-to-acknowledgment (MTTA) and mean-time-to-resolution (MTTR).
+
+To pull a sample report, use ``/workflow stats`` in any Mattermost channel:
+
+// TODO Add a screenshot - e.g. either re-use this screenshot or produce a new one: https://user-images.githubusercontent.com/1490756/65821266-4ad79d80-e201-11e9-8930-7beaa3dd0ed9.png
+
+The statistics are based on an aggregated summary of all individual workflow instances. You may also reset statistics at any time via ``/workflow reset-stats``.
+
+Note that you must be a System Administrator to execute these commands.
+
+Sample Schema
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Below is a full sample schema with a text trigger, four steps and multiple actions including transitions.
+
+For other sample schemas, `see here <// TODO Add other samples in https://github.com/mattermost/docs/tree/master/source/samples and link them here. See MM-18870>`_.
+
+  .. code-block:: json
```

Comment:
Alternatively link to https://gist.github.com/cpoile/09100e8ee79e5b4f26e837e6b3a69723